easybuild/easyconfigs/n/NVHPC/NVHPC-21.3.eb

```diff
@@ -0,0 +1,76 @@
+sources = [local_tarball_tmpl % '%(arch)s']
+checksums = [
+    {
+        local_tarball_tmpl % 'aarch64':
+            '88e0dbf8fcdd06a2ba06aacf65ae1625b8683688f6593ed3bf8ce129ce1b17b7',
+        local_tarball_tmpl % 'ppc64le':
+            '08cd0cd6c80d633f107b44f88685ada7f014fbf6eac19ef5ae4a7952cabe4037',
+        local_tarball_tmpl % 'x86_64':
+            'a3e3393040185ae844002fbc6c8eb4ffdfb97ce8b2ce29d796fe7e9a521fdc59'
+    }
+]
+
+local_gccver = '10.3.0'
+dependencies = [
+    ('GCCcore', local_gccver),
+    ('binutils', '2.36.1', '', ('GCCcore', local_gccver)),
+    # This is necessary to avoid cases where just libnuma.so.1 is present in the system and -lnuma fails
+    ('numactl', '2.0.14', '', ('GCCcore', local_gccver))
+]
+
+# specify default CUDA version that should be used by NVHPC
+# should match one of the CUDA versions that are included with this NVHPC version
+# (see install_components/Linux_x86_64/21.3/cuda/)
+# for NVHPC 21.3, those are: 11.2, 11.0, 10.2;
+# this version can be tweaked from the EasyBuild command line with
+# --try-amend=default_cuda_version="10.2" (for example)
+default_cuda_version = '11.2'
+
+# NVHPC EasyBlock supports some features, which can be set via CLI or this easyconfig.
+# The following list gives examples for the easyconfig
+#
+# NVHPC needs CUDA to work. Two options are available: 1) Use NVHPC-bundled CUDA, 2) use system CUDA
+# 1) Bundled CUDA
+#    If no easybuild dependency to CUDA is present, the bundled CUDA is taken. A version needs to be specified with
+#      default_cuda_version = "11.0"
+#    in this easyconfig file; alternatively, it can be specified through the command line during installation with
+#      --try-amend=default_cuda_version="10.2"
+# 2) CUDA provided via EasyBuild
+#    Use CUDAcore as a dependency, for example
+#      dependencies = [('CUDAcore', '11.0.2')]
+#    The parameter default_cuda_version still can be set as above.
+#    If not set, it will be deduced from the CUDA module (via $EBVERSIONCUDA)
+#
+# Define a NVHPC-default Compute Capability
+#   cuda_compute_capabilities = "8.0"
+# Can also be specified on the EasyBuild command line via --cuda-compute-capabilities=8.0
+# Only single values supported, not lists of values!
+#
+# Options to add/remove things to/from environment module (defaults shown)
+#   module_byo_compilers = False  # Remove compilers from PATH (Bring-your-own compilers)
+#   module_nvhpc_own_mpi = False  # Add NVHPC's own pre-compiled OpenMPI
+#   module_add_math_libs = False  # Add NVHPC's math libraries (which should be there from CUDA anyway)
+#   module_add_profilers = False  # Add NVHPC's NVIDIA Profilers
+#   module_add_nccl = False       # Add NVHPC's NCCL library
+#   module_add_nvshmem = False    # Add NVHPC's NVSHMEM library
+#   module_add_cuda = False       # Add NVHPC's bundled CUDA
+
+postinstallcmds = ['sed -i "s/LIBRARY_PATH/LLLIBRARY_PATH/" %(installdir)s/Linux_x86_64/21.3/compilers/bin/siterc']
```

Comment:
I think this deserves a comment to clarify why this is needed (and that it's a bug in NVHPC, not in our NVHPC easyblock).

Also, please use `%(version)s` rather than hardcoding `21.3`. And what about the hardcoded `x86_64` part, shouldn't we use `%(arch)s` instead? Or perhaps `/Linux_*/%(version)s/`?

Reply:
Yes, I fixed it now.